### PR TITLE
changes made to about.jsp

### PR DESCRIPTION
**Pull request by chenpionaar1998**
Practice for pull request :)

### CODE REVIEW

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -64,6 +77,8 @@
         This page should also be used to describe the features and improvements
         you've added.
       </p>
+
+
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Remove unnecessary lines.

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -34,7 +34,20 @@
 
   <div id="container">
     <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px; font-size: 15px;">
+
+      <h1>About Team 21 </h1>
+      <p>
+      Team 21 is a team of 3 university students from North America with the
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Minor indentation mismatches.  Please see  https://google.github.io/styleguide/htmlcssguide.html.

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -34,7 +34,20 @@
...
+       same passion of learning new skills and creating interesting apps. Here
+       is a list of the team members:
+     </p>
+      <ul>
+        <li><strong>Naomi McCracken :</strong>  </li>
+        <li><strong>Dean Chiu :</strong>  </li>
+        <li><strong>Sherry Yang :</strong>  </li>
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Unnecessary white space between </strong> and </li>

File: src/main/webapp/about.jsp [hunk 4]
```diff
@@ -34,7 +34,20 @@
 
   <div id="container">
     <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px; font-size: 15px;">
```
Comment (turtlefarmer) [inline src/main/webapp/about.jsp]: Please deploy any UI changes locally and attach a screenshot.  :)

**Review by turtlefarmer — CHANGES_REQUESTED** [review]

File: src/main/webapp/about.jsp (hunk 4 above)
Comment (chenpionaar1998) [reply src/main/webapp/about.jsp]: ui_change_font15px
ui_change_original

Comment (chenpionaar1998) [reply src/main/webapp/about.jsp]: I feel that the original font-size looks better, so I removed my changes for the font-size

Comment (turtlefarmer) [reply src/main/webapp/about.jsp]: Looks good thanks :)

**Review by turtlefarmer — APPROVED** [review]
All looks good.  Thanks :)

**Pull request merged**